include/open62541/server.h

```diff
@@ -1886,6 +1891,57 @@ typedef struct {
 UA_ServerStatistics UA_EXPORT
 UA_Server_getStatistics(UA_Server *server);
 
+/**
+  * Reverse Connect
+  * ---------------
+  *
```

Comment:
Make explicit that the connection status is for a SecureChannel.
Does connected mean we have the HEL/ACK handshake or does it mean the SecureChannel is open?
Not the session on top.